OracleWebLogic/dockerfiles/12.2.1.4/Dockerfile.developer

```diff
@@ -0,0 +1,88 @@
+#Copyright (c) 2014-2019 Oracle and/or its affiliates. All rights reserved.
+#
+#Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for WebLogic 12.2.1.4 Quick Install Distro
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) fmw_12.2.1.4.0_wls_quick_Disk1_1of1.zip
+#     Download the Developer Quick installer from http://www.oracle.com/technetwork/middleware/weblogic/downloads/wls-for-dev-1703574.html
+#
+# (2) server-jre-8uXX-linux-x64.tar.gz
+#     Download from http://www.oracle.com/technetwork/java/javase/downloads/server-jre8-downloads-2133154.html
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# Run:
+#      $ docker build -f Dockerfile.developer -t oracle/weblogic:12.2.1.4-developer .
+#
+# IMPORTANT
+# ---------
+# The resulting image of this Dockerfile contains a WLS Empty Domain.
+#
+# Pull base image
+# From the Oracle Registry
```

Comment:
This content is the same as the current 12.2.1.3 Dockerfiles

Reply:
Then the 12.2.1.3 Dockerfile is also wrong and should be fixed.

Reply:
Reached out to you privately (Slack).  I apologize as I reread the above because I sounded short.  Where did you find out about license content change?  Our other files are then wrong in this project and in about a dozen other projects.